openedx_learning/apps/authoring/contents/models.py

```diff
@@ -284,17 +298,34 @@ def mime_type(self) -> str:
 
     def file_path(self):
         """
-        Path at which this content is stored (or would be stored).
+        Logical path at which this content is stored (or would be stored).
+
+        This path is relative to OPENEDX_LEARNING['MEDIA'] configured storage
+        root.
+        """
+        return f"content/{self.learning_package.uuid}/{self.hash_digest}"
+
+    def storage_path(self):
+        """
+        The full OS path for the underlying file for this Content.
+
+        This will not be supported by all Storage class types.
+        """
+        return get_storage().path(self.file_path())
```

Comment:
Just noting that I found these names slightly confusing - I expected "storage_path" would be a path that works on any storage including cloud storage like s3, whereas "file_path" would be the "real os filesystem path" - but it's actually the reverse.

Perhaps `path` or `logical_path` instead of `file_path`, and `real_path` instead of `storage_path` if it's not too late to change. Not a big deal though.

Reply:
Renamed this to `path`/`os_path`, which will hopefully be clearer.